packages/react-native-windows-init/babel.config.js

```diff
@@ -0,0 +1,14 @@
+/**
```

Comment:
Do you need this file? I had it in override-tools so I could run `jest`, but I don't think we have anything using Babel here.